tests/unit/test_runners_helpers.py

```
@@ -0,0 +1,241 @@
+import unittest
+from unittest.mock import patch
+
+from secator.runners._helpers import (
+    run_extractors,
+    fmt_extractor,
+    extract_from_results,
+    parse_extractor,
+    process_extractor,
+    get_task_folder_id
+)
+from secator.output_types import OutputType, Url, Vulnerability, Target, Error
```

Ruff (0.11.9)

`secator.output_types.Error` imported but unused

Remove unused import: `secator.output_types.Error`

(F401)